pipeline/terraform/main.tf

```diff
@@ -62,6 +62,70 @@ module "catalogue_pipeline_20200717" {
   inferrer_model_data_bucket_name = aws_s3_bucket.inferrer_model_core_data.id
 }
 
+module "catalogue_pipeline_20200731" {
+  source = "./stack"
+
+  namespace = "catalogue-20200731"
+
+  release_label = "latest"
+
+  account_id      = data.aws_caller_identity.current.account_id
+  aws_region      = local.aws_region
+  vpc_id          = local.vpc_id
+  subnets         = local.private_subnets
+  private_subnets = local.private_subnets
+
+  dlq_alarm_arn = local.dlq_alarm_arn
+
+  # Transformer config
+  #
+  # If this pipeline is meant to be reindexed, remember to uncomment the
+  # reindexer topic names.
+
+  sierra_adapter_topic_arns = [
+    local.sierra_reindexer_topic_arn,
+    local.sierra_merged_bibs_topic_arn,
+    local.sierra_merged_items_topic_arn,
+  ]
+
+  miro_adapter_topic_arns = [
+    local.miro_reindexer_topic_arn,
```

Comment:
Are these meant to be uncommented?

Reply:
Yip as they'll be reading from the reindexer

Reply:
Just for initial reindex though right? Once its all done we have to remember to comment them again?

Reply:
Indeed, I'll swap that to prod when that's done, just wanted to get this in in case people are working on the pipeline, and don't want them to destroy it.

This deployment process is a little shaky, needs a rethink at some stage.